### PR TITLE
docs: clarify actions and syncs. add field mapping ui

### PR DESCRIPTION
**Pull request by tomkit**

### CODE REVIEW

File: docs/docs/integration-patterns/managed-syncs.md [hunk 1]
```diff
@@ -155,19 +155,27 @@ Sometimes, your customers may store data in non-standard fields (e.g. custom fie
 "schema": {
   "fields": [
     {
-      "name": "the_description",
-      "mapped_name": "Description"
+      "name": "description", // field in your system
```
Comment (asdfryan) [inline docs/docs/integration-patterns/managed-syncs.md:158]: FYI we've moved out the schema from the sync configuration. (I think our current documentation says it's part of the sync config)

**Review by lucasmarshall — APPROVED** [review]

File: docs/docs/integration-patterns/managed-syncs.md (hunk 1 above)
Comment (tomkit) [reply docs/docs/integration-patterns/managed-syncs.md:158]: Is it already on main? If not can we just land this and update when it changes?

Comment (asdfryan) [reply docs/docs/integration-patterns/managed-syncs.md:158]: Yes it's already on main.

File: docs/docs/integration-patterns/actions-api.md [hunk 2]
```diff
@@ -85,6 +87,9 @@ This feature is only supported for some destinations, not all.
 
 :::
 
+## Field mappings
+[Field mappings](../platform/field-mapping) can be used in the Actions API to reverse-map fields from your system to your customer's fields if you opted to use field mappings for Managed Syncs.
```
Comment (albertyfwu) [inline docs/docs/integration-patterns/actions-api.md:91]: do we need to say `reverse-map`? is `map` sufficient?

File: docs/docs/platform/field-mapping.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  ],
+  "allow_additional_field_mappings": false
+}
+...
+```
+
+**Example field mapping UI**:
+![field_mapping_ui](/img/field-mapping-ui.png 'salesforce field mapping ui')
+
+In this example, you want to map your customer's `Description` field to `description`, and you also want to map one of your customer's standard field `primary_address` and custom field to `revenue`. Through the [Update Schema endpoint](/api/v2/mgmt#tag/Schemas/operation/updateSchema), each of your customers can then map the appropriate revenue field to your `revenue` field and address to `primary_address`. At runtime, Supaglue will apply each customer's mapping and land the appropriate data into the `revenue` and `primary_address` column in your destination.
```
Comment (albertyfwu) [inline docs/docs/platform/field-mapping.md:35]: did you mean 

"map one of your customer's standard fields to `primary_address` and one of their custom fields to `revenue`"

?

Comment (albertyfwu) [reply docs/docs/platform/field-mapping.md:35]: "each of your customers can then map the appropriate revenue field to your `revenue` field and address field to `primary_address"

Comment (albertyfwu) [reply docs/docs/platform/field-mapping.md:35]: "land the appropriate data into the `revenue` and `primary_address` columns"

File: docs/docs/platform/field-mapping.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+The `allow_additional_field_mappings` flag is a way for individual customers to provide optional supplemental data not explicitly required by your schema, but that may be useful for your product. For example, extra attributes for filtering or features for ML models.
```
Comment (albertyfwu) [inline docs/docs/platform/field-mapping.md:37]: @asdfryan i think this is going away soon

**Pull request merged**